sentry/src/main/java/io/sentry/SentryTracer.java

```diff
@@ -452,16 +453,13 @@ private ISpan createChild(
     final @NotNull String operation = spanContext.getOperation();
     final @Nullable String description = spanContext.getDescription();
 
-    // TODO [POTEL] how should this work? return a noop? shouldn't block nested code from actually
-    // creating spans
-    //    if (SpanUtils.isIgnored(scopes.getOptions().getIgnoredSpanOrigins(),
-    // spanOptions.getOrigin())) {
-    //      return this;
-    //    }
+    if (SpanUtils.isIgnored(scopes.getOptions().getIgnoredSpanOrigins(), spanOptions.getOrigin())) {
```

Comment:
Hmm, this check could be moved up to the other ones returning a `NoOpSpan` (e.g. L:447 or L:451)